src/index.spec.js

```diff
@@ -28,6 +28,14 @@ const data = deepFreeze({
       username: 'Administrator',
       password: 'pas:sw@ord'
     }
+  },
+  NFS: {
+    string: 'nfs://192.168.100.225:media/nfs',
```

Comment:
That's probably something that should be fixed both in the parser and the server…